docker/Dockerfile-build-aggregator.jvm

```diff
@@ -1,42 +1,33 @@
-####
-# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
-###
-
-# Build the project
 FROM registry.access.redhat.com/ubi8/openjdk-11:latest AS build
 USER root
-COPY . /home/jboss
+COPY . /home/jboss/
```

Comment:
Does this trailing slash change something?

Reply:
doesn't make a difference.